lib/config/integrations/rails/railtie.rb

```diff
@@ -23,7 +23,7 @@ def preload
         # Development environment should reload settings on every request
         if ::Rails.env.development?
           initializer :config_reload_on_development do
-            ActionController::Base.class_eval do
+            ActiveSupport.on_load :action_controller_base do
```

Comment:
Can you dbl check which version is correct: yours or https://github.com/railsconfig/config/pull/258/files#diff-e1b40370ec751eed3246e103922adc0eR26 ?

Reply:
@amatsuda @Fryguy which version is better? This one or the one from #258?

Reply:
I think this version is better.  I don't think the class_eval is needed because on_load takes care of it being in the right context.

Reply:
Yeah, that's what I thought too! Then I will merge this one!